lib/tasks/gias_update.rake

```diff
@@ -0,0 +1,17 @@
+desc "Upsert GIAS data into GiasSchool"
+task gias_update: :environment do
+  logger = Logger.new($stdout)
+  today = Time.zone.today.strftime("%Y%m%d")
+  gias_filename = "edubasealldata#{today}.csv"
+  csv_path = Rails.root.join("tmp/#{gias_filename}")
+
+  logger.info "Downloading the new gias file for #{Time.zone.today}"
+  Down.download(
+    "#{ENV["GIAS_DOWNLOAD_LINK"]}/#{gias_filename}",
+    destination: csv_path
```

Comment:
Do we need to specify a destination? It looks like the [Down.download](https://github.com/janko/down#downloading) method will automatically save to a [Tempfile](https://rubyapi.org/o/Tempfile) – i.e. a File in a temporary location.

Reply:
Yeah, that's a good shout